_www.foia.gov-canary/request.html

```diff
@@ -175,15 +175,138 @@ <h1 class="usa-header-title">FOIA.gov</h1>
 
 <!-- End Header -->
 
-<div class="usa-grid" id="main">
+<div class="usa-grid-full" id="main">
```

Comment:
The canary is really meant for the legacy pages, maybe we should add this file to the ignore list in `.canaryignore`

Reply:
That seems smart -- I kind of wondered if it made sense for this or not! I'll add it to `.canaryignore` and remove the page from the canary folder.